client/lib/plans/features-list.js

```diff
@@ -1240,7 +1259,10 @@ export const FEATURES_LIST = {
 		getIcon: () => ( { icon: 'security', component: MaterialIcon } ),
 		getTitle: () =>
 			( {
-				i5: i18n.translate( 'Real-time Scan (automated)' ),
+				i5:
+					POPULAR_PRODUCTS_OFFERING_VARIANT === abtest( POPULAR_PRODUCTS_OFFERING_AB_TEST )
+						? i18n.translate( 'Scan (real-time, automated)' )
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Automated real-time scanning' )` *ES Score: 10*